cmd/helm/plugin_remove.go

```diff
@@ -60,7 +59,7 @@ func (pcmd *pluginRemoveCmd) complete(args []string) error {
 
 func (pcmd *pluginRemoveCmd) run() error {
 	debug("loading installed plugins from %s", settings.PluginDirs())
-	plugins, err := findPlugins(settings.PluginDirs())
+	plugins, err := findPlugins([]string{settings.PluginDirs()})
```

Comment:
What if someone tries to remove a plugin and it's only a system one? Wouldn't it not being found or removed be confusing? Also, how would people be expected to remove system wide plugins?

What about design that tells people when something is in the system and tells them it cannot be removed?  This is all to think about the UX. This same idea plays into install and update.